base/src/main/java/com/austindroids/austinfeedsme/common/events/EventsContract.kt

```diff
@@ -0,0 +1,29 @@
+package com.austindroids.austinfeedsme.common.events
+
+import com.austindroids.austinfeedsme.data.Event
+import com.austindroids.austinfeedsme.events.EventsFilterType
+
+/**
+ * Created by darrankelinske on 5/2/16.
+ */
+interface EventsContract {
+
+    interface View {
+        fun showEvents(events: List<Event>)
+        fun setPizzaCount(count: Int?)
```

Comment:
why are all those `Int?`? You should only use nullable types when you want `null` as a valid input

later you use `setBeerCount(if (yummyCounts[BEER.name] == null) 0 else yummyCounts[BEER.name])` etc so I assume they all shouldn't be nullable

Reply:
first off - thank you for the review! it was an artifact from automatic conversion - will fix up - thank you